third_party/katana/lib/usdKatana/readPoints.cpp

```diff
@@ -34,6 +34,49 @@ PXR_NAMESPACE_OPEN_SCOPE
 
 
 static FnKat::Attribute
+_GetPositionAttr(
+        const UsdGeomPoints& points,
+        const PxrUsdKatanaUsdInPrivateData& data)
+{
+    // Because of the logic used to gather these (in PxrUsdInOp::InitUsdInArgs),
+    // we can use the size of this vector as a simple test for whether motion
+    // blur is enabled, without needing access to the cook interface.
+    const std::vector<double>& motionSampleTimes =
+        data.GetUsdInArgs()->GetMotionSampleTimes();
```

Comment:
There is actually important logic in PxrUsdKatanaUsdInPrivateData::GetMotionSampleTimes() beyond what the UsdInArgs version does.  I think you should call it here on the "points" attr.  That said, there is some craziness regarding IsMotionBackwards() that we are trying to work through right now, so that we can get the PointInstancer and your new code doing the right thing in both forwards and backwards motion blur (we do backwards here, but want to make sure not to foist that decision on everyone).  I'll contact you again when we get it sorted, which should hopefully be soon.